cedar-policy/src/api.rs

```diff
@@ -712,6 +712,14 @@ impl Entities {
     pub fn write_to_json(&self, f: impl std::io::Write) -> std::result::Result<(), EntitiesError> {
         self.0.write_to_json(f)
     }
+
+    #[doc = include_str!("../experimental_warning.md")]
+    /// Visualize an `Entities` object in the graphviz `dot`
+    /// format. Entity visualization is best-effort and not well tested.
+    /// Feel free to submit an issue if you are using this feature and would like it improved.
+    pub fn to_dot_str(&self) -> std::result::Result<String, std::fmt::Error> {
+        self.0.to_dot_str()
```

Comment:
I wonder if `to_dot_str()` is implementable in using just our public API. It probably should be.

Reply:
Good question, I think it is. I'll leave that to a future refactor.